app/controllers/casa_admins_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CasaAdminsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :must_be_admin, only: [:new, :create]
+
```

Comment:
I suspect we may want to also have ` before_action :require_organization!` to ensure that the current user is part of this particular organization.

Adding this would also enable the use of the `current_organization` helper.